src/components/MarketDetail/index.js

```diff
@@ -192,21 +198,28 @@ class MarketDetail extends Component {
     const showWinning = market.oracle.isOutcomeSet
     const showLost = false // determine if we lost?
     const showWithdrawFees = this.props.defaultAccount && market.oracle.owner === this.props.defaultAccount
-
+    const timeToResolution = moment.utc(market.eventDescription.resolutionDate).local().diff(moment(), 'hours')
+    const oneWeekHours = 168
```

Comment:
Define this at the top of the file above the class declaration as `ONE_WEEK_IN_HOURS` or something like that